website/docs/guides/docs/docs-multi-instance.mdx

```diff
@@ -1,50 +1,45 @@
 ---
 id: multi-instance
-description: Use multiple docs plugin instances on a single Docusaurus site.
+description: Use the multi-instance feature to serve 2 or more doc sets.
 slug: /docs-multi-instance
 ---
 
 # Docs Multi-instance
 
-The `@docusaurus/plugin-content-docs` plugin can support [multi-instance](../../using-plugins.mdx#multi-instance-plugins-and-plugin-ids).
+The `@docusaurus/plugin-content-docs` plugin supports [multi-instance](../../using-plugins.mdx#multi-instance-plugins-and-plugin-ids). The multi-instance feature allows a site to serve 2 or more distinct sets of documentation from the same Docusaurus source.
 
 :::note
 
-This feature is only useful for [versioned documentation](./versioning.mdx). It is recommended to be familiar with docs versioning before reading this page. If you just want [multiple sidebars](./sidebar/multiple-sidebars.mdx), you can do so within one plugin.
+If you only want multiple sidebars, use a `sidebar` plugin instead. See [Using multiple sidebars](./sidebar/multiple-sidebars.mdx) for how to do this.
 
 :::
 
 ## Use-cases {#use-cases}
 
-Sometimes you want a Docusaurus site to host 2 distinct sets of documentation (or more).
+These are 2 of the everyday use cases for the multi-instance feature.
 
-These documentations may even have different versioning/release lifecycles.
+### Large, distinct documentation sets {#mobile-sdks-documentation}
 
-### Mobile SDKs documentation {#mobile-sdks-documentation}
+Your use case requires that your site serve 2 or more large documentation sets. For example, if you need to support a cross-platform mobile SDK, you may have 2 large documentation sets, such as:
 
-If you build a cross-platform mobile SDK, you may have 2 documentations:
+- Android SDK documentation
+- iOS SDK documentation
 
-- Android SDK documentation (`v1.0`, `v1.1`)
-- iOS SDK documentation (`v1.0`, `v2.0`)
-
-In this case, you can use a distinct docs plugin instance per mobile SDK documentation.
-
-:::caution
-
-If each documentation instance is very large, you should rather create 2 distinct Docusaurus sites.
+With large documentation sets, rebuilding may require extra work to verify and stage the set that did not change. By separating the two sets you can build only the set that changed.
 
-If someone edits the iOS documentation, is it really useful to rebuild everything, including the whole Android documentation that did not change?
```

Comment:
The previous advice was to create 2 separate Docusaurus sites, not using 2 separate Docusaurus instances. It's not the same advice anymore.

I don't know what "verify and stage the set that did not change" means. Docusaurus builds full sites, it does not "stage" documents nor can build a site partially